build.gradle missing compileOptions causes build error when compiling debug app

I'm getting the following build error when building a debug version of the app with `./gradlew assembleDebug assembleAndroidTest -DtestBuildType=debug` (this is to build a test version of the app for Detox).
```
> Task :react-native-wifi-reborn:mergeExtDexDebugAndroidTest FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':react-native-wifi-reborn:mergeExtDexDebugAndroidTest'.
> Could not resolve all files for configuration ':react-native-wifi-reborn:debugAndroidTestRuntimeClasspath'.
   > Failed to transform wifiutils-1.6.1.aar (com.thanosfisherman.wifiutils:wifiutils:1.6.1) to match attributes {artifactType=android-dex, dexing-enable-desugaring=false, dexing-is-debuggable=true, dexing-min-sdk=21, org.gradle.category=library, org.gradle.libraryelements=jar, org.gradle.status=release, org.gradle.usage=java-runtime}.
      > Execution failed for DexingNoClasspathTransform: /Users/petteriversen/.gradle/caches/transforms-2/files-2.1/8fa4caba63fb82d681eeaf5175b03313/jetified-wifiutils-1.6.1-runtime.jar.
         > Error while dexing.
           The dependency contains Java 8 bytecode. Please enable desugaring by adding the following to build.gradle
           android {
               compileOptions {
                   sourceCompatibility 1.8
                   targetCompatibility 1.8
               }
           }
           See https://developer.android.com/studio/write/java8-support.html for details. Alternatively, increase the minSdkVersion to 26 or above.

   > Failed to transform elvis-2.0.jar (com.thanosfisherman.elvis:elvis:2.0) to match attributes {artifactType=android-dex, dexing-enable-desugaring=false, dexing-is-debuggable=true, dexing-min-sdk=21, org.gradle.category=library, org.gradle.libraryelements=jar, org.gradle.status=release, org.gradle.usage=java-runtime}.
      > Execution failed for DexingNoClasspathTransform: /Users/petteriversen/.gradle/caches/transforms-2/files-2.1/f69a6d383859fdcfe652a01b3532ba21/jetified-elvis-2.0.jar.
         > Error while dexing.
```

The build error dissappears if I add `compileOptions` to `app/node_modules/react-native-wifi-reborn/build.gradle`:
```
android {
    ...

    compileOptions {
        sourceCompatibility JavaVersion.VERSION_1_8
        targetCompatibility JavaVersion.VERSION_1_8
    }
    
    ...
}
```
I'm not sure if this causes backwards compability issues, but [other repositories](https://github.com/auth0/react-native-auth0/pull/323/files) have added this as a fix to a [similar problem](https://github.com/auth0/react-native-auth0/issues/322) without doing a [breaking release](https://github.com/auth0/react-native-auth0/releases/tag/v2.6.0).

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -16,6 +16,11 @@ android {
     compileSdkVersion 29
     buildToolsVersion '29.0.3'
 
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+
     defaultConfig {
         minSdkVersion 21
         targetSdkVersion 29

